models/lido/liquidity/base/lido_liquidity_base_aerodrome_cl_pools.sql

```diff
@@ -0,0 +1,319 @@
+ , wsteth_prices_hourly AS (
+    SELECT distinct
+        DATE_TRUNC('hour', minute) time, 
+        contract_address as token,
+        symbol,
+        decimals,
+        last_value(price) over (partition by DATE_TRUNC('hour', minute), contract_address ORDER BY  minute range between unbounded preceding AND unbounded following) AS price
+    FROM {{source('prices','usd')}} p
+     {% if not is_incremental() %}
+    WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
+     {% else %}
+    WHERE DATE_TRUNC('day', p.minute) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+     {% endif %}
+    and blockchain = 'base' and contract_address = 0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452
+    
+) 
+
+, wsteth_prices_hourly_with_lead AS (
+select time, 
+       lead(time, 1, date_trunc('hour', now() + interval '1'  hour)) over (order by time) as next_time, 
+       price
+from wsteth_prices_hourly
+)
+
+, mint_events AS ( 
+ select DATE_TRUNC('day', m.evt_block_time) AS time,
+      m.contract_address AS pool,
+      cr.token0,
+      cr.token1,
+      SUM(CAST(amount0 AS DOUBLE)) AS amount0,
+      SUM(CAST(amount1 AS DOUBLE)) AS amount1
+ from {{source('aerodrome_base','CLPool_evt_Mint')}} m
+ left join {{source('aerodrome_base','CLFactory_evt_PoolCreated')}} cr on m.contract_address = cr.pool 
+ 
+ {% if not is_incremental() %}
+ WHERE DATE_TRUNC('day', m.evt_block_time) >= DATE '{{ project_start_date }}'
+ {% else %}
+ WHERE DATE_TRUNC('day', m.evt_block_time) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+ {% endif %}
+ 
+ and m.contract_address in (select address from pools)
+ group by 1,2,3,4
+ )
+
+  , collect_events as (
+ select DATE_TRUNC('day', b.evt_block_time) AS time,
+      b.contract_address AS pool,
+      cr.token0,
+      cr.token1,
+      (-1)*SUM(CAST(amount0 AS DOUBLE)) AS amount0,
+      (-1)*SUM(CAST(amount1 AS DOUBLE)) AS amount1
+ from {{source('aerodrome_base','CLPool_evt_Collect')}} b
+ left join {{source('aerodrome_base','CLFactory_evt_PoolCreated')}} cr on b.contract_address = cr.pool 
+ 
+ {% if not is_incremental() %}
+ WHERE DATE_TRUNC('day', b.evt_block_time) >= DATE '{{ project_start_date }}'
+ {% else %}
+ WHERE DATE_TRUNC('day', b.evt_block_time) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+ {% endif %}
+ 
+ and b.contract_address in (select address from pools)
+ group by 1,2,3,4
+ 
+ )
+
+, swap_events as (
+ select DATE_TRUNC('day', s.evt_block_time) AS time,
+      s.contract_address AS pool,
+      cr.token0,
+      cr.token1,
+      SUM(CAST(amount0 AS DOUBLE)) AS amount0,
+      SUM(CAST(amount1 AS DOUBLE)) AS amount1
+ from {{source('aerodrome_base','CLPool_evt_Swap')}} s
+ left join {{source('aerodrome_base','CLFactory_evt_PoolCreated')}} cr on s.contract_address = cr.pool
+ 
+ {% if not is_incremental() %}
+  WHERE DATE_TRUNC('day', s.evt_block_time) >= DATE '{{ project_start_date }}'
+ {% else %}
+  WHERE DATE_TRUNC('day', s.evt_block_time) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
```

Comment:
Maybe we can use incremental predicate for incremental filters:
```suggestion
  WHERE {{ incremental_predicate('s.evt_block_time') }}
```